docs/guide.md

```diff
@@ -0,0 +1,210 @@
+# Getting Started with PatternPack
+
+## Creating a New PatternPack
+The idea behind PatternPack is to extract the styles from your application and move them into a separate repository where they can be shared between one or multiple projects.
+
+You have multiple methods to generate a Pattern Library:
+
+- Build your own configuration (covered in this section)
+- Start with the [example library](https://github.com/patternpack/patternpack-example-library) ([documentation](https://github.com/patternpack/patternpack/blob/master/docs/getting-started.md#start-with-the-example-pattern-library))
+- Use Yeoman Generator ([documentation](https://github.com/patternpack/patternpack/blob/master/docs/getting-started.md#use-the-yeoman-generators))
+
+If you're starting with the example library or Yeoman, go ahead and skip to [the next section](LINK).
+
+The first step is to initialize a new project with a Git repository: create a new directory and run `git init`.
+
+### Project Dependencies
+Once your new project is created, you'll need to get a few dependencies set up:
+
+- `npm init` and follow the prompts to generate a node project
+- `npm install grunt patternpack --save-dev` to install grunt and PatternPack as dependencies
+
+At this point, we've got project dependencies ready.
+
+### Basic Configuration
+PatternPack is a grunt plugin, so you'll need to create a `gruntfile.js` in your project's root directory and add PatternPack to it:
+
+#### gruntfile.js
+```js
+module.exports = function (grunt) {
+  grunt.initConfig({
+    patternpack: {
+      run: {},
+      build: {},
+      release: {}
+    }
+  });
+
+  grunt.loadNpmTasks('patternpack');
+
+  grunt.registerTask('default', ['patternpack:run']);
+}
+```
+
+This grunt configuration exposes three tasks to your project:
+
+- `patternpack:run` which starts up the development environment
+- `patternpack:build` which generates the web server files
+- `patternpack:release`, which exposes the versioning and release tasks
+
+The last line also maps the default `grunt` command to generate your pattern library and start up a development environment.
+
+### Advanced Configuration
+PatternPack provides smart defaults, however it can be extensively configured to suit your needs. Here is what all the configurable PatternPack options look like with all options set to default:
```

Comment:
We should direct the users to the main documentation for the advanced configuration options.  We don't want to maintain this in multiple places.